examples/hardhat/deployments/optimism/solcInputs/a4e2764a77fec30ccb674d800c46e436.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
can we remove this too?

Reply:
you mean add this file to the .gitignore? Yes, that would be useful

Reply:
deleted and updated .gitignore